tests/jest.config.e2e.ts

```diff
@@ -185,7 +185,7 @@ export default {
 	// ],
 
 	// Changing default timeout from 5000 to 30000 for tests
-	testTimeout: 300000,
+	testTimeout: 180000,
```

Comment:
We set this to be higher for the scaffolding tests running in the pipeline. They run a lot slower there which causes the test to timeout and report a failure